### ISSUE TITLE
JAWS doesn't read out diff viewer icons

### ISSUE THREAD

**Issue by milemons** [microsoft/vscode#107021]
When viewing the diff viewer with JAWS screen reader, JAWS will read out all the lines in the editor but not the line numbers or the icons for addition or deletion. As a result, customers who depend on screen readers will have no context on whether a line was deleted or added during review.

**monaco-editor version:** all - repro'd on the editor playground, vscode, and Azure DevOps pull requests
**Browser:** Chrome 84
**OS:** Windows
**Playground code that reproduces the issue:** https://microsoft.github.io/monaco-editor/

**Comment by alexdima** [comment]
Since the editor ships the exact same diff editor as vscode, moving to the vscode repo.

**Comment by alexdima** [comment]
@milemons For the diff editor, we have the diff review experience (`F7`) that is built especially for screen readers. That experience resembles how `git diff` outputs on the command line. Can you please try that out, does that work for you?
